pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -212,7 +214,7 @@
         "4.0.1",
         "4.3.0"
```

Comment:
You need to also add the stable package versions otherwise the baselines will fail.